src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -143,7 +143,10 @@ def trigger_dbt_cli_command_flow():
     # write the profile if overwrite or no profiles exist
     if overwrite_profiles or not Path(profiles_path).expanduser().exists():
         if dbt_cli_profile is None:
-            raise ValueError("Provide `dbt_cli_profile` keyword for writing profiles")
+            raise ValueError(
+                f"Profile not found. Provide `dbt_cli_profile` or\
+                             ensure profiles.yml exists at {profiles_path}."
+            )
```

Comment:
nit: This string usage will ensure no gaps between the lines when printing the error message.
```suggestion
            raise ValueError(
                "Profile not found. Provide `dbt_cli_profile` or"
                f" ensure profiles.yml exists at {profiles_path}."
            )
```